cmd/gindoid/genhtml.go

```diff
@@ -34,6 +41,26 @@ func mkhtml(cmd *cobra.Command, args []string) {
 		fmt.Printf("Using server at %s\n", ginurl)
 	}
 
+	doibase := libgin.ReadConf("doibase")
+	if doibase == "" {
+		fmt.Printf("Using default DOI prefix: %s\n", defdoibase)
+		doibase = defdoibase
+	}
+
+	var storeurl *url.URL
+	storeurlstr := libgin.ReadConf("storeurl")
+	if storeurlstr == "" {
+		fmt.Printf("Using default store URL: %s\n", defstoreurl)
+		storeurlstr = defstoreurl
```

Comment:
`storeurlstr` is not used later on. This and the next line could simply be merged to `storeurl, _ = url.Parse(defstoreurl)`